src/main.rs

```diff
@@ -19,17 +23,24 @@ async fn main() {
     };
 
     let mut client = get_lnd_client(args).expect("failed to connect");
+    let mut client_clone = client.clone();
```

Comment:
The `Lightning`/`Peers`/`Signer` clients already implement `clone()`, so you can use `client.lightning().clone` instead of requiring a further upstream change to `tonic_lnd`. 

Took me a minute to figure out how `tonic` is dealing with this all / why everything needs to be mutable, seems like it's unavoidable and best practise is to just [clone](https://github.com/carlaKC/lndk/pull/20#issuecomment-1466952243).